tests/test_datasets/test_internal_datasets.py

```diff
@@ -73,10 +73,16 @@ def test_not_present_part():
         _ = load_dataset("custom_internal_dataset", parts="val")
 
 
-@pytest.mark.skip(reason="Dataset is too large for testing in GitHub.")
 @pytest.mark.parametrize(
     "dataset_name, expected_shape, expected_min_date, expected_max_date",
-    [("electricity_15T", (140256, 370), pd.to_datetime("2011-01-01 00:15:00"), pd.to_datetime("2015-01-01 00:00:00"))],
+    [
+        ("m4_hourly", (1008, 414), pd.to_datetime("2021-11-20 01:00:00"), pd.to_datetime("2022-01-01 00:00:00")),
```

Comment:
I think we should specify why we should have such spaces at least something like that `(308 + 700)`  where 700 is horizon and check test shape too